src/alignment.hpp

```diff
@@ -21,7 +21,7 @@ class Alignment {
 
   // Is the alignment non-empty and do all sequences have the same length?
   bool IsValid() const;
-  std::string at(const std::string& taxon) const;
+  const std::string& at(const std::string& taxon) const;
```

Comment:
The same change should be made to the definition in alignment.cpp

Reply:
🤦 fixed